flink-formats/flink-protobuf/src/test/java/org/apache/flink/formats/protobuf/ProtobufSQLITCaseTest.java

```diff
@@ -328,4 +331,33 @@ public void testSinkWithNullLiteralWithEscape() throws Exception {
         MapTest.InnerMessageTest innerMessageTest = mapTest.getMap2Map().get("b");
         assertEquals(MapTest.InnerMessageTest.getDefaultInstance(), innerMessageTest);
     }
+
+    @Test
+    public void testUnsupportedBulkFilesystemSink() {
+        env().setParallelism(1);
+        String sql =
+                "create table bigdata_sink ( "
+                        + "	a int, "
+                        + "	map1 map<string,string>,"
+                        + " map2 map<string, row<a int, b bigint>>"
+                        + ")  with ("
+                        + "	'connector' = 'filesystem', "
+                        + "	'path' = '/tmp/unused', "
+                        + "	'format' = 'protobuf', "
+                        + " 'protobuf.message-class-name' = 'org.apache.flink.formats.protobuf.testproto.MapTest'"
+                        + ")";
+        tEnv().executeSql(sql);
```

Comment:
I would have preferred to throw the exception here, but I didn't find the right hook into the connector.